Source/items.cpp

```diff
@@ -2030,6 +2251,12 @@ void PrintItemMisc(const Item &item)
 		strcpy(tempstr, _("Right-click to view"));
 		AddPanelString(tempstr);
 	}
+	if (item._iMiscId == IMISC_SOULSTONE) {
+		strcpy(tempstr, _("Contains Diablo's essence"));

```

Comment:
Maybe the description could give a hint for the player to only use it at level 16. Otherwise, it could surprise players.

Reply:
Removing the part that kills players. Going to make the item unusable entirely unless the player is on level 16 and the Diablo quest has been completed.

Reply:
I like that. Maybe it could play something like:
```
myPlayer.Say(HeroSpeech::ICantCastThatHere);
```

Reply:
Selected "That won't work here" for levels other than 16 and "I can't use this, yet" for level 16 when the quest hasn't been completed. These voice lines seemed more relevant.

Reply:
@kphoenix137 perfect!